docs/EpistemicConstraints.md

```diff
@@ -145,7 +145,7 @@ Is satisfied if `field` is a number less than or equal to `value`.
 
 Is satisfied if `field` has at least the [granularity](./DataTypes.md#Granularity) specified in `value`.
 
-## Temporal constraints
+## Datetime constraints
```

Comment:
As above, i would still refer to temporal constraints, but maybe include a comment to say - for now - this only relates to the datetime data type.